plugin-libs/plugin-command/src/test/java/com/salesforce/bazel/eclipse/command/mock/MockCommandBuilder.java

```diff
@@ -159,7 +159,7 @@ public void addSimulatedOutput(String nameForLog, List<String> outputLines, List
      * @param matchers one or more matchers that will specifically target a command
      */
     public void addSimulatedOutput(String nameForLog, List<String> outputLines, List<String> errorLines, List<MockCommandSimulatedOutputMatcher> matchers) {
-        MockCommandSimulatedOutput out = new MockCommandSimulatedOutput(nameForLog, outputLines, errorLines);
+        MockCommandSimulatedOutput out = new MockCommandSimulatedOutput(nameForLog, outputLines, errorLines, matchers);
```

Comment:
this was the bug, matchers param was not passed along. this allowed commands to be matched regardless of the command line args. when the bug was fixed, tests broke because they were expecting the old style launching scheme 'bazel run' instead of invoking the launcher script bazel-bin/a/b/myservice/myservice